distbench_engine.cc

```diff
@@ -1091,15 +1093,13 @@ void DistBenchEngine::RunAction(ActionState* action_state) {
           copied_server_rpc_state->have_dedicated_thread = true;
           copied_server_rpc_state->SetFreeStateFunction(
               [=] { delete copied_server_rpc_state; });
-          RunRegisteredThread(
```

Comment:
For my understanding, why did you remove this?

Reply:
The old code created a new thread unconditionally, which has high overhead. The new code uses a threadpool interface to try to recycle threads.